lib/zssAuth.js

```diff
@@ -7,11 +7,20 @@
   
   Copyright Contributors to the Zowe Project.
 */
+const Promise = require('bluebird');
+const ipaddr = require('ipaddr.js');
+const url = require('url');
+const makeProfileNameForRequest = require('./safprofile').makeProfileNameForRequest;
+
+const log = global.COM_RS_COMMON_LOGGER.makeComponentLogger("_unp.auth.zss");
+
+const DEFAULT_CLASS = "ZOWE";
 
 const DEFAULT_EXPIRATION_MS = 3600000 //hour;
 
 function ZssAuthenticator(pluginDef, pluginConf, serverConf) {
   this.authPluginID = pluginDef.identifier;
```

Comment:
I think the instance ID should be expected within serverConf.
I think that the id can come from the top-level config, rather than node.id and agent.id, because I'm not sure what the expected behavior would be if the zlux instance did not match with the zss instance.
I expect this id to be missing for now, until the community determines where this is even coming from.

Reply:
So what is the conclusion? Just read it from `serverConf` for now and don't bother actually putting anything there?  (will always be undefined)

Reply:
serverConf.instanceId that comes from zluxserver.json instanceId... not within any inner object like node or agent